src/App.js

```diff
@@ -1,25 +1,43 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react'
+import mockBigCats from './mockBigCats'
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch
+} from 'react-router-dom'
+import Header from './components/Header'
+import Footer from './components/Footer'
+import NotFound from './pages/NotFound'
+import Home from './pages/Home'
+import BigCatEdit from './pages/BigCatEdit'
+import BigCatIndex from './pages/BigCatIndex'
+import BigCatNew from './pages/BigCatNew'
+import BigCatShow from './pages/BigCatShow'
+import './App.css'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+export class App extends Component {
```

Comment:
Last export to remove
```suggestion
class App extends Component {
```

Reply:
done